readme.md

```diff
@@ -44,6 +44,33 @@ This is a live Twitter feed of any tweets matching the search term
 `#dartconf OR #flutterio OR #angulardart`. Selecting a tweet will open the corresponding
 item directly at twitter.com. A pull-down gesture will refresh the tweet data.
 
+## Build the app for development
+
+To build the app in release mode for Android, you will need to provide the following files:
+
+ * `assets/app.token.txt` Containing Twitter app credentials
+ * `android/app/google-services.json` A Firebase project config file
+
+### Creating the Twitter access token
+First create a [Twitter app](https://apps.twitter.com/)
+
+This will create a `Consumer key` and a `Consumer secret`. 
```

Comment:
Perhaps instead point to the relevant twitter docs pages? I think very few people will need to actually create a functioning `app.token.txt`, and the below is highly twitter specific.

Or, perhaps we could move much of this readme out into a `developing.md`.

Reply:
If it gets much longer than this we might want to move it out. If you have this, you basically have a running app.